README.md

```diff
@@ -103,32 +133,33 @@ same. There are three major stages, each with its own steps.
       button, Storyblok will make a request to Vercel to start a preview
       deployment.
    3. Look for deployment notifications in Slack. Vercel will send a message to
```

Comment:
I had to do a little digging. 

For the Labs Storyblok space, the published event is connected to a webhook on Vercel that deploys the GitHub `develop` branch. But for the Consulting Storyblok space, that same field is blank. (Oddly, the "workflow changed" event is connected to a webhook that no longer exists in the Consulting Vercel project, so I deleted it.)

Do you remember why we turned this off for the Consulting Storyblok space?

Reply:
We specifically disabled that webhook for Consulting. When we're editing existing pages on Consulting, we do ***not*** want to have to Publish something in order to kick off a preview build:

- We do *not* want to Publish the thing we're working on and want to create a preview build for, because by definition it's still not ready to go live. *And*, we have no easy way of reverting the Published version to what it was before.
- It is clunky and non-intuitive to have to go to some other piece of content in Storyblok and Publish _that_, in order to get a preview build.

It's much more intuitive to have a Task named "trigger preview build".

Reply:
And yeah, that "workflow changed" hook should have been deleted. That was left over from the workflow from before your overhaul.